sources/packages/backend/apps/api/src/testHelpers/fake-entities/education-program-offering-fake.ts

```diff
@@ -29,5 +30,6 @@ export function createFakeEducationProgramOffering(
   offering.courseLoad = 45;
   offering.hasOfferingWILComponent = "no";
   offering.offeringDeclaration = true;
+  offering.offeringStatus = OfferingStatus.CreationPending;
```

Comment:
I feel that looking at the overall purpose of this factory, offering status must be defaulted to approved, with supporting data. IMO Creation pending offering is only useful for  those tests which check the initial status of the offering and otherwise we always would need an approved offering.